recipes/_install.rb

```diff
@@ -29,16 +29,16 @@ def nsp
 include_recipe 'cerner_splunk::_restart_marker'
 
 # Actions
-# This service definition is used only for ensuring splunk is started during the run
-service 'splunk-start' do
+# This service definition is used for ensuring splunk is started during the run and to stop splunk service
+service 'splunk' do
   service_name service
   action :nothing
-  supports status: true, start: true
+  supports status: true, start: true, stop: true
   notifies :delete, 'file[splunk-marker]', :immediately
```

Comment:
Actually I think we do want to delete it.  On the off chance that config changed somewhere else on the search head during the run we wouldn't want the delayed service restart action to get kicked off after the service was stopped.